deploy/helm/sumologic/values.yaml

```diff
@@ -202,7 +202,7 @@ sumologic:
 
     ## Option to control the enabling of metadata filter plugin cache_size. 
     ## ref: https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter#configuration
-    bearerCacheSize: "1000"
+    bearerCacheSize: "10000"
```

Comment:
where did `bearer` come from here? Can we change it to just be `cacheSize` (I'm assuming no one's touched this yet).

Reply:
same with `bearerCacheTtl` below.